temporalnexus/operation.go

```diff
@@ -52,6 +56,18 @@ import (
 	"go.temporal.io/sdk/workflow"
 )
 
+// tokenSource is used as the "source" operation token header.
+const tokenSource = "io.temporal.sdk"
+
+// workflowRunOperationTokenType is used as the "type" operation token header for workflow run operations.
+const workflowRunOperationTokenType = "WorkflowRunOperation"
+
+// workflowRunOperationToken is the decoded form of the workflow run operation token.
+type workflowRunOperationToken struct {
```

Comment:
Why not the run ID as well?

Reply:
We generate the operation token before we have a run ID because we send it with the start request callback headers for complete-before-start.